example/pubspec.yaml

```diff
@@ -1,13 +1,14 @@
 name: signalr_flutter_example
 description: Demonstrates how to use the signalr_flutter plugin.
-version: 1.0.0+1
+version: 1.1.0+1
 
 # The following line prevents the package from being accidentally published to
 # pub.dev using `flutter pub publish`. This is preferred for private packages.
-publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+publish_to: "none" # Remove this line if you wish to publish to pub.dev
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  flutter: ^3.19.0
+  sdk: ^3.2.0
```

Comment:
Do the same for SDK also. Do not constraint upper bound, constraint lower bound.